azure-pipelines.yml

```diff
@@ -19,7 +19,12 @@ variables:
     _HelixSource: official/aspnet/AspNetCore-Tooling/$(Build.SourceBranch)
     _SignType: real
     _SignArgs: /p:DotNetSignType=$(_SignType) /p:TeamName=$(_TeamName)
-    _OfficialBuildIdArgs: /p:OfficialBuildId=$(Build.BuildNumber)
+    _OfficialBuildIdArgs: /p:OfficialBuildId=$(Build.BuildNumber) 
+      /p:ManifestTeamProject=AspNet
+      /p:ManifestTeamName=AspNetCore-Tooling
+      /p:ManifestBuildBranch=$(Build.SourceBranch)
+      /p:ManifestBuildNumber=$(Build.BuildNumber)
+      /p:ManifestPublishUrl=https://vsdrop.corp.microsoft.com/file/v1/Products/AspNet/AspNetCore-Tooling/$(Build.SourceBranch)/$(Build.BuildNumber);
```

Comment:
@natemcmaster - I think we could clean this up a bit more in the future. One of the challenges we have here is that we have to produce a file during the build that says where we *will upload the VSIX in the future*.

Reply:
I think some others do this in two steps. But we aren't as smart as they are.